root/docker-mods

```diff
@@ -250,11 +262,22 @@ run_mods() {
     done
 }
 
-# Tamper check legacy services
-tamper_check
+# Main script loop
+
+SCRIPTS_DIR_OLD="/config/custom-cont-init.d"
+SCRIPTS_DIR="/custom-cont-init.d"
+SERVICES_DIR_OLD="/config/custom-services.d"
+SERVICES_DIR="/custom-services.d"
```

Comment:
Let's keep these at the very top of the file (like they are here https://github.com/linuxserver/docker-baseimage-alpine/blob/e109ae692e276e4005375bcf4404880c4a89abbf/root/etc/s6-overlay/s6-rc.d/init-custom-files/run#L4-L6 )